CHANGELOG.md

```diff
@@ -1,5 +1,85 @@
 # Changelog
 
+## [0.20.0] - 2023-05-28
+**Breaking**
+- (Windows) The Windows installer version of Sunshine is now always launched by the Sunshine Service. Manually launching Sunshine.exe from Program Files is no longer supported. This was necessary to address security issues caused by non-admin users having access to Sunshine's config data. If you have set up Task Scheduler or other mechanisms to launch Sunshine automatically, remove those from your system before updating.
+- (Windows) The Start Menu shortcut has been redesigned for use with the Sunshine Service. It now launches the Web UI instead of starting another instance of Sunshine if one is already running. The Start Menu shortcut is now the recommended method for opening the Web UI and launching Sunshine.
+- (UPnP) If the Moonlight Internet Hosting Tool is installed alongside Sunshine, you must remove it or upgrade to v5.6 or later to prevent conflicts with Sunshine's UPnP support. As a reminder, the Moonlight Internet Hosting Tool is not required to stream over the Internet with Sunshine. Instead, simply enable UPnP in the Sunshine Web UI.
```

Comment:
```suggestion
- (Network/UPnP) If the Moonlight Internet Hosting Tool is installed alongside Sunshine, you must remove it or upgrade to v5.6 or later to prevent conflicts with Sunshine's UPnP support. As a reminder, the Moonlight Internet Hosting Tool is not required to stream over the Internet with Sunshine. Instead, simply enable UPnP in the Sunshine Web UI.
```

P.S. Should we put an entry in the docs that this tool is not necessary when using Sunshine? https://docs.lizardbyte.dev/projects/sunshine/en/nightly/gamestream/gamestream.html

Reply:
Good idea, added one to that page.